code/validation/covid19.py

```diff
@@ -100,17 +100,24 @@ def covid19_row_validator(column_index_dict, row, codes):
         except ValueError:
             pass  # ignore, caught by `json_io_dict_from_quantile_csv_file()`
 
-    # 3. validate forecast_date and target_end_date date formats
+    # 3. validate forecast_date and target_end_date
     forecast_date = row[column_index_dict['forecast_date']]
     target_end_date = row[column_index_dict['target_end_date']]
+    
+    # 3.1 Expect date formats
     forecast_date = _parse_date(forecast_date)  # None if invalid format
     target_end_date = _parse_date(target_end_date)  # ""
-
     if not forecast_date or not target_end_date:
         error_messages.append(f"Error > invalid forecast_date or target_end_date format. forecast_date={forecast_date!r}. "
                               f"target_end_date={target_end_date}. row={row}")
-        return error_messages  # terminate - depends on valid dates
-
+        return error_messages
+
+    # 3.2 Expect a minimum 5 days between forecast_date and target_end_date
+    date_diff = target_end_date - forecast_date
+    if date_diff < 5:
```

Comment:
This causes
<img width="594" alt="image" src="https://user-images.githubusercontent.com/1156307/110027038-64034880-7d29-11eb-8cc0-43d14019efc6.png">
I think you might need `date_diff.days` or something like that.

Reply:
Ah thanks, silly mistake! Fixed.